packages/slate-react/src/utils/environment.ts

```diff
@@ -43,6 +43,14 @@ export const IS_FIREFOX_LEGACY =
 export const IS_QQBROWSER =
   typeof navigator !== 'undefined' && /.*QQBrowser/.test(navigator.userAgent)
 
+// UC mobile browser
+export const IS_UC_MOBILE =
+  typeof navigator !== 'undefined' && /.*UCBrowser/.test(navigator.userAgent)
+
+// Wechat browser
+export const IS_WECHATBROWSER =
+  typeof navigator !== 'undefined' && /.*Wechat/.test(navigator.userAgent)
```

Comment:
/.*Wechat/i is better  becouse of  Enterprise WeChat.

Reply:
Please open a new PR, as conversations in closed PRs generally don't get noticed. Thanks!